### DISCUSSION TITLE
How to port latest version of filament into #Daz3d?

### DISCUSSION THREAD

**Discussion by not16** [google/filament#4245] [Q&A]
[Daz3d Filament Page](https://www.daz3d.com/filament-information-page)

I can't got back, but I'm wondering if I can implement a newer build of filament instead of waiting for them to launch a new version of Daz3d. 

Does anyone know how to do this?

**Comment by romainguy** [comment]
I'm not sure how that could be done without access to the source code of Daz3D.
